### PR TITLE
Guide for IntelliJ support

### PR DESCRIPTION
**Pull request by mwl**

### CODE REVIEW

**Comment by mwl** [comment]
CLA has been signed

**Comment by mwl** [comment]
I just realised that this should probably have gone into the [ide-tools](https://github.com/toitware/ide-tools) repo instead. Do you want me to move it over?

**Comment by kasperl** [comment]
Re-checking CLA status: @cla-bot check.

File: README.md [hunk 1]
```diff
@@ -201,6 +201,25 @@ In the VS Code extension (version 1.3.7+) set the `toitLanguageServer.command` s
 
 This makes the extension use the language server that was compiled in the [build step](#build-for-host-machine).
 
+### Jetbrains IntelliJ Platform
```
Comment (kasperl) [inline README.md:204]: Suggestion: Jetbrains' IntelliJ platform

The other titles in this README do not use title case, so for consistency I don't think this one should either.

File: README.md [hunk 2]
```diff
@@ -201,6 +201,25 @@ In the VS Code extension (version 1.3.7+) set the `toitLanguageServer.command` s
...
+Applies to distributions like IntelliJ IDEA, GoLand, CLion, PHPStorm etc, but has not been tested with Fleet.
+
+---
+
+Through the plugin system, IntelliJ has limited support for LSP. In particular the [LSP Support](https://plugins.jetbrains.com/plugin/10209-lsp-support) plugin has been verified to work with Toit's LSP server.
+
+With the plugin installed, a server definition needs to be added, in Preferences > Languages & Frameworks > Language Server Protocol, with the following parameters:
+* `Executable`
+* Extension: `toit`
+* Path: _Path to toit executable_ i.e. `/usr/local/bin/toit`
```
Comment (kasperl) [inline README.md:218]: In context of this README, I think this should point to `$TOIT_SDK/bin/toitlsp` with a note that if you've built this yourself, you can find it in `build/host/sdk/bin/toitlsp`. No arguments should be necessary.

**Review by kasperl — COMMENTED** [review]
Thank you for your contributions! Here are a few comments:

File: README.md (hunk 2 above)
Comment (floitsch) [reply README.md:218]: Actually, it needs the same arguments as for the "Other IDEs": `--toitc=$TOIT_SDK/bin/toitc`.

**Comment by floitsch** [comment]
I agree that the README will get cluttered pretty fast if we add a new section for each IDE, and VIM support is only in the ide-tools repository: https://github.com/toitware/ide-tools/tree/master/vim
I'm fine with committing for now, but this will probably move to a Wiki at some point.

**Comment by mwl** [comment]
Thanks guys. Really appreciate your input. I will incorporate your suggestions and move it to the ide-tools repository ASAP.

**Pull request closed without merging**